src/input.scss

```diff
@@ -10,16 +10,17 @@ $block: #{$fd-namespace}-input;
 .#{$block} {
   @include fd-form-text();
   @include fd-var-size("height", $fd-forms-height, --fd-forms-height);
+  @include fd-var-size("line-height", $fd-forms-height, --fd-forms-height);
 
   width: 100%;
 
   &::placeholder {
-    @include fd-reset();
```

Comment:
This should probably not be removed, just adding what you needed to override.

Reply:
OK. I've reimplemented the `fd-reset()` and have added overrides to `line-height` in subsequent CSS rules.